main.js

```diff
@@ -0,0 +1,61 @@
+const books = document.getElementById('books');
+let titulo = document.getElementById('titulo');
+let autor = document.getElementById('autor');
+const addBtn = document.getElementById('btn');
+
+let dataBooks = [];
+
+if (localStorage.getItem('book')) {
+  dataBooks = JSON.parse(localStorage.getItem('book'));
+}
+
+class Libro {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+function deleteBook(index, libro) {
+  libro.remove();
+  dataBooks.splice(index, 1);
+  localStorage.setItem('book', JSON.stringify(dataBooks));
+}
+
+const printList = () => {
+  books.innerHTML = null;
+  if (localStorage.getItem('book')) {
+    dataBooks = JSON.parse(localStorage.getItem('book'));
+  }
+  for (let i = 0; i < dataBooks.length; i += 1) {
+    const bookStorage = document.createElement('div');
+    const bookName = document.createElement('p');
+    const authorName = document.createElement('p');
+    const deleteBtn = document.createElement('button');
+    bookName.textContent = dataBooks[i].title;
+    authorName.textContent = dataBooks[i].author;
+    deleteBtn.textContent = 'Delete';
+    books.appendChild(bookStorage);
+    bookStorage.appendChild(bookName);
+    bookStorage.appendChild(authorName);
+    bookStorage.appendChild(deleteBtn);
+    deleteBtn.addEventListener('click', () => {
+      deleteBook(i, bookStorage);
+    });
+  }
+};
+
+printList();
+
+const addBook = (title, author) => {
+  const libro = new Libro(title, author);
+  dataBooks.push(libro);
+  localStorage.setItem('book', JSON.stringify(dataBooks));
+  printList();
+};
```

Comment:
## Optional
- [ ] I suggest that you use a unique `id` property for each book since it would make it easier to remove books with duplicate values.

Reply:
- [ ] Consider using `concat, slice, filter` methods that don't mutate the array instead of `push, pop, splice`